lib/debezium/decimal.go

```diff
@@ -62,33 +62,6 @@ func decodeBigInt(data []byte) *big.Int {
 	return bigInt
 }
 
-// decimalWithNewExponent takes a [apd.Decimal] and returns a new [apd.Decimal] with a the given exponent.
-// If the new exponent is less precise then the extra digits will be truncated.
-func decimalWithNewExponent(decimal *apd.Decimal, newExponent int32) *apd.Decimal {
```

Comment:
Moved this to https://github.com/artie-labs/transfer/pull/765/files#diff-05c5ea56a4927faef9a74d525d6312544af5a191a532c2d8cc2d9fd4ee59ecefR113 to get around circular imports, no changes.